### PR TITLE
Formatted desktop version

### PR DESCRIPTION
**Pull request by emyrue**
- Added media queries
- Added images
- Added menu items
- Used display: grid

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -51,20 +65,30 @@ <h1 class="project-heading">Projects</h1>
...
+        <img class="more-dots" alt="dots" src="./images/dots.png">
+        <img class="semicircle" alt="shapes" src="./images/semicircle.png">
+        <img class="two-squares" alt="shapes" src="./images/twosquares.png">
+        <img class="diamond" alt="shape" src="./images/diamond.png">
+        <img class="shapes" alt="shapes" src="./images/shapes.png">
+        <img class="yellow-tear" alt="shape" src="./images/yelllowtear.png">
 
         <!--Project 1-->
 
         <section class="project project1">
```
Comment (gemmen29) [inline index.html:81]: Please try to match the Figma design in width 993px for the projects and about sections.
| Your design | Figma design |
| --- | --- |
| screencapture-raw-githack-emyrue-Portfolio-form-index-html-2022-02-17-01_25_45 | Template 8-main |

And in width 992px the image that takes class `banner2` disappear and this is how the page looks like:
screencapture-raw-githack-emyrue-Portfolio-form-index-html-2022-02-17-01_38_25

File: index.html [hunk 2]
```diff
@@ -208,10 +234,11 @@ <h1 class="skills-heading">Skills</h1>
...
-            <p class="contact-paragraph">Lorem ipsum dolor sit amet,
-                consectetur adipiscing elit,
-                sed do eiusmod tempor incididunt
-                ut labore et dolore magna.</p>
+            <p class="contact-paragraph">Lorem ipsum dolor sit amet, 
+                consectetur adipiscing elit, sed do eiusmod tempor incididunt 
+                ut labore et dolore magna aliqua. Ut enim ad minim veniam, 
+                quis nostrud exercitation ullamco laboris nisi ut aliquip ex 
+                ea commodo consequat. </p>
             <form class="form" action="https://formspree.io/f/mjvlwyao" method="POST">
```
Comment (gemmen29) [inline index.html:242]: Please try to match the Figma design in a mobile version for the email input
| Your design | Figma design |
| --- | --- |
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @emyrue, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, @emyrue,

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

## [OPTIONAL] Before merging, please fix the space after the footer in the desktop version. 
Screen Shot 2022-02-17 at 21 16 19


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**